### ISSUE TITLE
WSL 2 - Cannot create new project

### ISSUE THREAD

**Issue by djboboch** [livebud/bud#38]
Been using go for a while and have been looking for a framework like bud for a while.

I installed bud using the install script, then when trying to create a new project using `bud create hello` the installed runs some things happen however the cli get stuck ( see screenshot).

![image](https://user-images.githubusercontent.com/26084619/168428954-a852840a-3460-4f87-9cf9-5a7d8b345da8.png)

No new folders appear inside of the directory and the terminal is stuck.

**Comment by matthewmueller** [comment]
Could you send an ABRT signal to the bud process when it's hanging and share the stack trace? https://pro-tips-dot-com.tumblr.com/post/47677612115/kill-a-hung-go-process-and-print-stack-traces. That may help us narrow down the problem.

**Comment by matthewmueller** [comment]
I think this should be solved in [v0.1.1](https://github.com/livebud/bud/releases/tag/v0.1.1). Could you try again?

To upgrade, run:

```sh
curl -sf curl https://raw.githubusercontent.com/livebud/bud/main/install.sh | sh
```

**Comment by djboboch** [comment]
I can confirm `bud create` now works with WLS2. I am closing the issue.
